### PR TITLE
SVCS-2648 Added unauthenticated request changes to the migration guide

### PR DESCRIPTION
**Pull request by chad-pelly**

### CODE REVIEW

File: MIGRATING_TO_V8.0.0.MD [hunk 1]
```diff
@@ -207,3 +207,12 @@ These are the affected properties, listed by the relevant SDK method:
...
 - `exh.templates.resolveAsJsonUsingCode`
   - Response: `ALL KEYS`
+
+## Unauthenticated requests
+For all [unauthenticated methods](#unauthenticated-methods), in v8.0.0 the response from these requests will now correctly undergo the same transformation steps as any other request.
+
+Noticeably their timestamps previously returned as strings are now converted to date objects.
+
+### Unauthenticated Methods
+- `exh.auth.authenticate`
```
Comment (wwaaijer-exh) [inline MIGRATING_TO_V8.0.0.MD:217]: This and auth.confirmMfa too right? Also just for oAuth1 if I recall correctly 🤔

File: MIGRATING_TO_V8.0.0.MD [hunk 2]
```diff
@@ -207,3 +207,12 @@ These are the affected properties, listed by the relevant SDK method:
   - Response: `ALL KEYS`
 - `exh.templates.resolveAsJsonUsingCode`
   - Response: `ALL KEYS`
+
+## Unauthenticated requests
+For all [unauthenticated methods](#unauthenticated-methods), in v8.0.0 the response from these requests will now correctly undergo the same transformation steps as any other request.
+
+Noticeably their timestamps previously returned as strings are now converted to date objects.
```
Comment (wwaaijer-exh) [inline MIGRATING_TO_V8.0.0.MD:214]: Nice! Can we also mention we've adjusted the types accordingly? So they have an extra trigger to look for

File: MIGRATING_TO_V8.0.0.MD [hunk 3]
```diff
@@ -207,3 +207,12 @@ These are the affected properties, listed by the relevant SDK method:
   - Response: `ALL KEYS`
 - `exh.templates.resolveAsJsonUsingCode`
   - Response: `ALL KEYS`
+
+## Unauthenticated requests
+For all [unauthenticated methods](#unauthenticated-methods), in v8.0.0 the response from these requests will now correctly undergo the same transformation steps as any other request.
```
Comment (chad-pelly) [inline MIGRATING_TO_V8.0.0.MD:212]: BTW the anchors provided in the section above this in this file are wrong. 😞 

###-affected-properties -> #affected-properties. Didn't want to make this change in this PR for clarity. But I can do it before we merge this? 🤔

File: MIGRATING_TO_V8.0.0.MD (hunk 1 above)
Comment (chad-pelly) [reply MIGRATING_TO_V8.0.0.MD:217]: YES!

File: MIGRATING_TO_V8.0.0.MD [hunk 4]
```diff
@@ -207,3 +207,13 @@ These are the affected properties, listed by the relevant SDK method:
...
+
+## Unauthenticated requests
+For all [unauthenticated methods](#unauthenticated-methods), in v8.0.0 the response will now correctly undergo the same transformation steps as any other request.
+
+Noticeably their timestamps previously returned as strings are now converted to date objects, the types of these methods have been updated to reflect this change.
+
+### Unauthenticated Methods
+- `exh.auth.authenticate`
+- `exh.users.createAccount`
+- `exh.auth.confirmMfa` for OAuth1 clients
```
Comment (chad-pelly) [inline MIGRATING_TO_V8.0.0.MD:219]: Although it's technically more correct how would we feel about removing `for OAuth1 clients`?

It's kinda ugly and well the OAuth2 version doesn't return timestamps anyways... 🤷

**Review by AaronTrippaers — DISMISSED** [review]

File: MIGRATING_TO_V8.0.0.MD (hunk 4 above)
Comment (wwaaijer-exh) [reply MIGRATING_TO_V8.0.0.MD:219]: Also okay for me! Can you move this line to be next to the `authenticate` then though? Sorry for the nitpicking 😅

**Review by AaronTrippaers — APPROVED** [review]

**Pull request merged**